.github/workflows/stale-issues-and-prs.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Mark and close stale issues and identify stale pull requests'
+# This workflow was adapted from the uberon stale-issue workflow: https://github.com/obophenotype/uberon/blob/master/.github/workflows/stale-issues.yml
+# See https://github.com/actions/stale for info about the action
+
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          days-before-issue-stale: 630
+          days-before-issue-close: 30
+          close-issue-label: 'autoclosed-unfixed'
+          stale-issue-message: 'This issue has not seen any activity in the past 2 years. It will automatically 30 days from now if no action is taken.'
```

Comment:
```suggestion
          stale-issue-message: 'This issue has not seen any activity in the past 2 years. It will be closed automatically 60 days from now if no action is taken.'
```